modules/tests/src/test/scala/dev/profunktor/redis4cats/OptimisticLockSuite.scala

```diff
@@ -35,7 +35,7 @@ class OptimisticLockSuite extends IOSuite {
   private val InitialValue = "a"
   private val UpdatedValue = "b"
 
-  test("Optimistic lock allows single update") {
+  test("Optimistic lock allows single update".ignore) {
```

Comment:
This needs re-enabling 🙂

```suggestion
  test("Optimistic lock allows single update") {
```